content/negotiation/quest_cards.lua

```diff
@@ -384,6 +384,31 @@ local CARDS = {
             end)
         end,
     },
+    faction_negotiation_hinder =
+    {
+        name = "Faction Hinder",
+        show_dealt = false,
+        quip = "support",
```

Comment:
The quip here seems wrong. It should be hinder because it's a hinder.
Probably my fault because I forgot to change it when copying, and since there is already custom quips, I didn't notice any issue. Can you change this and the wrong quip I have in the other card?